### ISSUE TITLE
SOD a bazillion errors

### ISSUE THREAD

**Issue by nwardoh** [Rottenbeer/ItemRack#271]
When clicking the minimap button the errors just start scrolling non-stop. 

![image](https://github.com/user-attachments/assets/7d0b6310-c6da-4a85-a81c-c167faa2c607)

**Comment by Tegas** [comment]
The [GetMouseFocus](https://warcraft.wiki.gg/wiki/API_GetMouseFocus) function has been replaced by [GetMouseFoci](https://warcraft.wiki.gg/wiki/API_GetMouseFoci).

https://warcraft.wiki.gg/wiki/Patch_11.0.0/API_changes

I got it working by swapping 
	`local frame = GetMouseFocus()`
to 
	`local frame = GetMouseFoci()[1]`

**Comment by Tegas** [comment]
It seems there's already a PR with a better fix. 
https://github.com/Rottenbeer/ItemRack/pull/270/commits/fe393625d77df777115171f6c05c1374f2f8cc5c

**Comment by konffen** [comment]
Managed to copypaste the error from BugSack:

```
4442x ItemRack/ItemRack.lua:1289: attempt to call global 'GetMouseFocus' (a nil value)
[string "@ItemRack/ItemRack.lua"]:1289: in function `func'
[string "@ItemRack/ItemRack.lua"]:1016: in function `OnUpdate'
[string "*ItemRack.xml:7_OnUpdate"]:1: in function <[string "*ItemRack.xml:7_OnUpdate"]:1>

Locals:
(*temporary) = nil
(*temporary) = "attempt to call global 'GetMouseFocus' (a nil value)"
```

**Comment by nwardoh** [comment]
What's the plan here? This game is basically unplayable as a tank without this addon

**Comment by mikinho** [comment]
Have a fix made https://github.com/Rottenbeer/ItemRack/pull/270

Can provide a zip for those who need it until it is merged

**Comment by Rottenbeer** [comment]
Released as 4.11
